Makefile

```diff
@@ -1,8 +1,8 @@
-## This Makefile is meant to be run on Unix hosts - as such, it only supports the few
-## operations that don't require on a Windows host, mostly code generation and linting.
+os=windows
+CMDS=server
+all: build test-go
 
-.DEFAULT_GOAL := all
-SHELL := /bin/bash
+include release-tools/build.make
```

Comment:
Can you add a short comment why you are including this? I guess it's for the tests, but it would be better to be explicit about that.

Reply:
added comments. we include release-tools for building binary and testing

Reply:
added it.